system/netdata.service.v235.in

```diff
@@ -18,7 +18,7 @@ StateDirectoryMode=0755
 CacheDirectoryMode=0755
 LogsDirectoryMode=2750
 EnvironmentFile=-/etc/default/netdata
-ExecStart=/usr/sbin/netdata -D $EXTRA_OPTS
+ExecStart=@sbindir_POST@/netdata -D $EXTRA_OPTS
```

Comment:
I guess `EnvironmentFile` should be updated too.

Reply:
Nope, that’s correct, although it’s Debian-specific.